content/pages/home.json

```diff
@@ -6,8 +6,8 @@
   "blocks": [
     {
       "_template": "hero",
-      "headline": "Unlock your Next.js project with live editing",
-      "subline": "Empower your teammates to build and edit content stored in your Git repository.",
+      "headline": "Empower Your Team with Visual Editing",
+      "subline": "Tina allows teams to build faster and deliver to production with confidence.",
```

Comment:
When reading this out of context, one may wonder what are we building and delivering?

....to build **websites** faster and deliver **content** to production with confidence

Reply:
```suggestion
      "subline": "Build visual editing on top of content versioned in GitHub",
```